oci-umount.conf

```diff
@@ -6,3 +6,6 @@
 /var/lib/docker-latest/overlay
 /var/lib/docker-latest/devicemapper
 /var/lib/docker-latest/containers
+/var/lib/container/storage/overlay2
+/var/lib/container/storage/overlay
+/var/lib/container/storage/overlay-containers
```

Comment:
These are created by containers/storage.  Although overlay2 is going away.  I think there is only going to be overlay and overlay-containers, eventually we will have lvm and lvm-containers.

@nalind PTAL